docs/src/1.1.BasicExample.mdx

```diff
@@ -1,7 +1,7 @@
-import { BasicExample } from './jsx/allLiveEditors.js'
+import { BasicExample } from './jsx/allLiveEditors'
 
 # Basic Example
 
-A simple example to show how to set up Worldview to render simple shapes.
+A simple example to show how to set up `Worldview` to render simple shapes.
```

Comment:
I don't think we should add this everywhere, Worldview is like a proper noun and it looks messy if we have to use `` ` `` all the time